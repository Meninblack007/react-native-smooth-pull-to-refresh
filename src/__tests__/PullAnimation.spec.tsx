import {shallow} from "enzyme";
import React from "react";
import {Animated} from "react-native";
import {PullAnimation, PullAnimationProps} from "../PullAnimation";

describe("PullAnimation spec", () => {
  it("App show PullAnimation", () => {
    const props: PullAnimationProps = {
      yValues: {from: -50, to: 10},
      isRefreshing: false,
      minPullDistance: 70,
      scrollY: new Animated.Value(0),
    };
    const wrapper = shallow(<PullAnimation {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
