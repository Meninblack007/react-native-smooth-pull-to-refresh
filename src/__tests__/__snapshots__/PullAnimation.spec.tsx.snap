// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PullAnimation spec App show PullAnimation 1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "position": "absolute",
        "top": null,
      },
    ]
  }
/>
`;
